votes/0137-bufferflies-as-pd-committer.md

```diff
@@ -0,0 +1,11 @@
+# A Vote for bufferflies as PD Committer
+
+## Proposal
+
+[@bufferflies](https://github.com/bufferflies) has been working on `tikv/pd` for more than four months. He has continuously contributed to scheduler、recover、statistics for PD. His pr are listed [here]( https://github.com/tikv/pd/pulls?q=+is%3Apr+author%3Abufferflies+).
```

Comment:
```suggestion
[@bufferflies](https://github.com/bufferflies) has been working on `tikv/pd` for more than four months. He has continuously contributed to scheduler, recover, statistics for PD. His pull requests are listed [here]( https://github.com/tikv/pd/pulls?q=+is%3Apr+author%3Abufferflies+).
```

@nolouch I'd suggest use English punctuation, and if possible, describe more on why we nominate @bufferflies as committer to build an image to the community.